raw_data/preprocessed/africa/ethiopa_english.txt

```diff
@@ -0,0 +1,82 @@
+
+Grievance Procedures
+(17) [1] Any person dissatisfied with the authorization or monitoring or any decision of the Authority or the relevant regional environmental agency regarding the project may submit a grievance notice to the head of the Authority or the relevant regional environmental agency, as may be appropriate. [2] The decision of the head of the Authority or relevant regional environmental agency shall, as provided under Sub Article [1] above, be issued within 30 days following the receipt of the grievance. 
+
+PART SIX
+
+Offences and Penalties
+(18) [1] Without prejudice to the provisions of the Penal Code, any person who violates the provision of this Proclamation or of any other relevant law or directive commits an offence and shall be liable accordingly. [2] Any person who, without obtaining authorization from the Authority or the relevant regional environmental agency, or makes false presentations in an environmental impact assessment study report commits an offence and shall be liable to a fine of not less than fifty thousand Birr and not more than one hundred thousand Birr. [3] Any person commits an offence if he fails to keep records or to fulfill conditions of authorization issued pursuant to this Proclamation and shall be liable to a fine of not less than ten thousand Birr and not more than twenty thousand Birr. [4] When a juridical person commits an offence, in addition to whatever penalty it may be meted with, the manager who failed to exercise all due diligence shall be liable to a fine of not less than five thousand Birr and not more than ten thousand Birr. [5] The court before which a person is prosecuted for an offence under this Proclamation or regulations or directives emanating from it, may, in addition to any penalty it may impose, order the convicted person to restore or in any other way compensate for the damage inflicted. 
+
+PART SEVEN - MISCELLANEOUS PROVISIONS
+
+Power to Issue regulations
+(19) The Council of Ministers may issue Regulations necessary for the effective implementation of this Proclamation.
+
+Power to Issue Directives
+(20) The Authority may issue directives necessary for the effective implementation of this Proclamation.
+
+Duty to Cooperate
+(21) Any person shall have the duty to cooperate in the implementation of this Proclamation.
+
+Laws
+(22) Any law or practice inconsistent with this Proclamation is inapplicable regarding matters provided herein.
+
+Effective Date
+(23) This Proclamation shall come into force as of the 3rd day of December, 2002. Done at Addis Ababa, this 3rd day of December, 2002.
```

Comment:
I doubt that the second sentence (i.e. `Done at ...`) is part of the article. In all of the other legislation that I have reviewed, that sentence has been associated with the signatures.

Reply:
Good point, I have removed it.